server.mjs

```diff
@@ -152,27 +158,27 @@ MinifyRoom().then(function(result) {
     }
   });
 
-  app.get('/s/:link/:junk', function(req, res, next) {
-    if(!sharedLinks[`/s/${req.params.link}`])
+  router.get('/s/:link/:junk', function(req, res, next) {
+    if(!sharedLinks[`${config.HTTP_URL_PREFIX}/s/${req.params.link}`])
```

Comment:
Personally, I wouldn't use the prefix for saving links so they still work when changing the server prefix.

Reply:
Agreed.